src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+    <p className="Person__age">
+      I am
+      {' '}
+      {person.age}
+    </p>
+    )}
+    <p className="Person__partner">
+      {person.isMarried ? `${person.partnerName} is my ` : `I am not married`}
+      {person.sex === 'm' && person.isMarried ? `wife` : ''}
+      {person.sex === 'f' && person.isMarried ? 'husband' : ''}
```

Comment:
decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```

Reply:
Still not fixed from the previous review 
<img width="900" alt="image" src="https://github.com/mate-academy/react_person/assets/64750958/de67353d-4310-4b0f-93a4-c9cc8e9f3a33">